### PR TITLE
Removes suggested search pop-up and adds other year buttons in results

### PR DESCRIPTION
**Pull request by schalkventer**
image

image

### CODE REVIEW

**Comment by jbothma** [comment]
remember we talked about maybe making the colour a grey because green is a bit too strong in this context? Is that still an option or did you decide to just go with green for now?

I think the order of the years should be reversed - this order doesn't match my expectation and is the opposite of the year selector.

File: _includes/components/header-and-footer/Search/index.jsx [hunk 1]
```diff
@@ -55,18 +55,6 @@ export default function SearchMarkup(props) {
               }}
             />
           </div>
-
-          <ResultsArea
```
Comment (jbothma) [inline _includes/components/header-and-footer/Search/index.jsx]: Can the implementation of ResultsArea be removed now?

**Comment by schalkventer** [comment]
@jbothma, tried grey, but green seemed more fitting since it's a call to action. I was concerned that someone will scroll to the end of the results or see the box when there are no results and not realise that it is a call to action for more content at first glance.

File: _includes/components/header-and-footer/Search/index.jsx (hunk 1 above)
Comment (schalkventer) [reply _includes/components/header-and-footer/Search/index.jsx]: Just to confirm, ResultsArea is the suggested dropdown that shows as you are typing. If I remember correctly the consensus was that we will remove it for now.

**Comment by jbothma** [comment]
Agreed. That's why i was surprised that i didn't see the component code
getting removed. Figured you'd forgotten after removing the call to it.

**Comment by schalkventer** [comment]
@jbothma Ok I see. Yeah, I'm keeping the component in the repo for now (it doesn't get compiled into production if it is not called), since I figured we might possibly revisit it and change the behaviour at one point. However for now I've just disabled it.

**Pull request merged**